app/controllers/gobierto_participation/processes_controller.rb

```diff
@@ -1,16 +1,17 @@
+# frozen_string_literal: true
+
 module GobiertoParticipation
   class ProcessesController < GobiertoParticipation::ApplicationController
-
     def index
-      @processes = current_site.processes.process
-      @groups    = current_site.processes.group_process
+      @processes = current_site.processes.open
```

Comment:
@apradillap you need to filter and show only published processes, and not processes in draft